librz/core/agraph.c

```diff
@@ -4362,18 +4347,6 @@ RZ_API int rz_core_visual_graph(RzCore *core, RzAGraph *g, RzAnalysisFunction *_
 				agraph_toggle_callgraph (g);
 			}
 			break;
-		case 'Z':
```

Comment:
Actually.. Is there anything wrong with the folding on the ascii-graph? I think it works well enough... or not?

Reply:
@ret2libc Doesn't work see:

<img width="290" alt="image" src="https://user-images.githubusercontent.com/1408600/102404411-fd9f4180-3fe7-11eb-84f3-fd95dddd4d65.png">

Reply:
Did you press z while in graph mode or while in visual mode?

Reply:
while in graph mode

Reply:
ahhh sorry, I was doing the `z`, not `Z`. Ok then!